Sources/StreamChatUI/CommonViews/ChatInputView/ChatMessageInputView.swift

```diff
@@ -36,10 +36,9 @@ open class _ChatMessageInputView<ExtraData: ExtraDataTypes>: _View, ComponentsPr
     }
     
     override open func setUpLayout() {
-        embed(container)
+        embed(container, insets: .init(top: 1, leading: 4, bottom: 1, trailing: 0))
```

Comment:
The preferred way of doing this is to pin the view to the layout margins and change the margins. It makes it easier to change the layout just by changing the margins constants rather than adding new constraints.

In fact, the `inset` parameter in `embed` should be probably removed because it doesn't have a good use case.

Reply:
✅

Reply:
I did not remove that functionality from the `embed` function btw, because it is being used in a lot of places, and it breaks the UI in multiple places. So I think it should be a ticket on its own. It would be nice to have the same API, but change the `embed` function to add the constraints to the layout margins, and the inset should be applied to the superview.layoutMargins instead